### PR TITLE
Fix markdown hyperlink 

### PR DESCRIPTION
**Pull request by ejoreo**

### CODE REVIEW

File: content/project-status.md [hunk 1]
```diff
@@ -2,26 +2,47 @@
...
+### 0.3
+This release is expected to contain a functional metrics implementation. Again, you may see changes to the API and the SDK after this point, but it's anticipated that these releases will allow you to explore and understand the changes that have been made from OpenCensus. In addition, this release is anticipated to include improvements to OpenTelemetry Context, various semantic conventions around trace and metric metadata, and a finalized OpenTelemetry protocol.
 
-## 0.4
-This release is expected to contain further improvements to the tracing and metrics API and SDK in response to feedback gathered during the prior alpha releases. Additionally, we expect that the OpenTelemetry protocol will be fully implemented and languages will include an exporter to the OpenTelemetry Collector. At this phase, we anticipate all APIs will be unlikely to change.
+### Beta
```
Comment (flands) [inline content/project-status.md]: This potentially makes it look like `beta` is a version when 0.4 will be first beta and 0.5 will be second. Perhaps use `#### 0.4 & 0.5` and add alpha section above for 0.2 and 0.3?

Comment (ejoreo) [reply content/project-status.md]: According to the [release plan](https://docs.google.com/document/d/1eviggoIguOS89dgKL-8ntUOCfPP6FLWAoQkGoBDsld0/edit?pli=1): "We’ve set the following requirements for components to reach beta: All spec version <= 0.3 features must be implemented. For APIs and SDKs this includes all 0.3 features for traces, metrics, context propagation, and resource metadata." 

It's been my impression v0.3 is the beta benchmark.  I can combine the sections: '### 0.3 & beta' as a single header if you think that will be clearer.

Comment (MikeGoldsmith) [reply content/project-status.md]: I think it makes sense to combine 0.3 and beta sections as they are linked - maybe something like `0.3 - Beta`.

Comment (flands) [reply content/project-status.md]: sorry -- this is what I meant ^

**Comment by ejoreo** [comment]
@austinlparker I think this is g2g

File: content/project-status.md [hunk 2]
```diff
@@ -2,26 +2,45 @@
...
+### 0.2
+This release included a functional tracing implementation. While the API may change, you should be able to use these releases to begin exploring and understanding the API and the SDK, and the changes that have been made from OpenTracing/OpenCensus. This release also included support for distributed context using the W3C TraceContext specification.
 
-## 0.3
-This release is expected to contain a functional metrics implementation. Again, you may see changes to the API and the SDK after this point, but it's anticipated that these releases will allow you to explore and understand the changes that have been made from OpenCensus. In addition, this release is anticpated to include improvements to OpenTelemetry Context (such as handling baggage), various semantic conventions around trace and metric metadata, and a finalized OpenTelemetry protocol.
+### 0.3 & beta
```
Comment (mtwo) [inline content/project-status.md]:
```suggestion
### 0.3 (first beta release)
```

File: content/project-status.md [hunk 3]
```diff
@@ -2,26 +2,45 @@
...
+We’ve set the following requirements for components to reach beta:
+
+* All spec version <= 0.3 features must be implemented. For APIs and SDKs this includes all 0.3 features for traces, metrics, context propagation, and resource metadata. For the Collector this includes the OpenTelemetry proto format
+* Smaller changes are already scheduled for the v0.4 and v0.5 versions of the spec, but they may be implemented during the beta. Presumably, beta feedback will create further changes
+* All APIs will attempt to be final, with the goal of not introducing any breaking changes between beta and GA RC. If breaking changes must be introduced between beta and GA RC, they will be small
+* Components must support the OpenTelemetry-native exporter. Components should support exporters for Jaeger, Prometheus, and Zipkin, though some of these might be added soon after the component enters beta. The Collector must also include receivers for these formats
+* APIs must include at least one HTTP and gRPC integration, though these are packaged separately. APIs should include at least one SQL integration, and can include a web framework integration (this is a stretch goal)
+
+
+### 0.4 & 0.5
```
Comment (mtwo) [inline content/project-status.md]:
```suggestion
### 0.4 (additional beta release)
```

**Comment by austinlparker** [comment]
Could you fix the merge conflict and then I can merge this? Thanks

**Pull request merged**